docs/Contributing.asciidoc

```diff
@@ -687,3 +689,24 @@ $ make test
 
 And if you need code examples, there are some plugins
 https://github.com/os-autoinst/openQA/tree/master/lib/OpenQA/WebAPI/Plugin[included with openQA].
+
+== Checking for JavaScript problems
+One can use the tool +jshint+ to check for problems within JavaScript code. It can be installed
```

Comment:
Sounds like something we want in the Makefile in the next step

Reply:
Annoyingly jshint is not packaged and the easiest way to get it is by letting npm install it in a custom directory. So one would still need to set e.g. an environment variable to provide the path of jshint.

Reply:
Sure. You could still add it as a custom make target and look for its existence, e.g.

```
make test-jshint:
    command -v jshint 2>/dev/null || (print "Need jshint" && exit 1)
    jshint …
```